packages/fonts/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@heycar-uikit/fonts",
+  "version": "1.0.0",
+  "description": "HeyCar-UIKit fonts",
+  "main": "dist/index.js",
+  "module": "./dist/esm/index.js",
```

Comment:
how I would be doing it.
set here
```
"main": "path/to/default.css",
```
remove  `"module": "./dist/esm/index.js",` - it is not needed absolutely.

remove bold.css and normal.css, move 200|300|500.css to the root.

add the documentation to the storybook without sandbox.

that's all.